### ISSUE TITLE
Wrong version file in 2.1.1 download

### ISSUE THREAD

**Issue by HebaruSan** [steamp0rt/Kerbalism#306]
2.1.1 says it is 2.0.1:

https://spacedock.info/mod/1774/Kerbalism/download/2.1.1

```json
{
  "NAME": "Kerbalism",
  "URL": "http://ksp-avc.cybutek.net/version.php?id=559",
  "DOWNLOAD": "https://spacedock.info/mod/1774/Kerbalism",
  "CHANGE_LOG_URL": "https://spacedock.info/mod/1774/Kerbalism",
  "VERSION": {"MAJOR": 2, "MINOR": 0, "PATCH": 1, "BUILD": 0},
  "KSP_VERSION": {"MAJOR": 1, "MINOR": 6, "PATCH": 1},
  "KSP_VERSION_MIN": {"MAJOR": 1, "MINOR": 3, "PATCH": 1},
  "KSP_VERSION_MAX": {"MAJOR": 1, "MINOR": 6, "PATCH": 99}
}
```

**Comment by GJNelson7030** [comment]
Thanks! Wasn't sure I was reporting this in correct manner. First time for me.

**Comment by PiezPiedPy** [comment]
been fixed with PR #304 for the next release
